src/items/external-blocks.md

```diff
@@ -29,9 +29,12 @@
 > _NamedFunctionParametersWithVariadics_ :\
 > &nbsp;&nbsp; ( _NamedFunctionParam_ `,` )<sup>\*</sup> _NamedFunctionParam_ `,` `...`
 
-External blocks form the basis for Rust's foreign function interface.
-Declarations in an external block describe symbols in external, non-Rust
-libraries.
+External blocks provide _declarations_ of items that are not _defined_ in the
+current crate and are the basis of Rust's foreign function interface. Using
```

Comment:
"Using" is ambiguous; for example:
```rust
extern "C" {
    fn foo();
}

fn bar() { let _ = foo; }
```
does compile just fine and is one form of "using".

Reply:
Nice catch. I would suggest replacing "using" with "calling" for functions and "accessing" for statics, but I'm not sure if those are comprehensive enough.

Reply:
I agree that "calling" and "accessing" are better. :)

Reply:
Note that for functions this can be explained by the type: they are `unsafe fn`s, whereas I think even `let _ = FFI_STATIC;` requires `unsafe`.

Reply:
@eddyb `let _ = FFI_STATIC;` works without `unsafe`; https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=5e582a3d7aa9dc49b3a250a6861a80b9

Reply:
I've disambiguated that.